src/content/hr/engine/animation-data/readme.md

```diff
@@ -0,0 +1,39 @@
+The [hr-ek] uses several types of animations when building a model_animation_graph from source data. Using legacy commands these are filetypes, however in the Sidecar pipeline animation types are defined in the sidecar.xml
```

Comment:
is that a valid way to add a link?

Reply:
Link works but it should have a more descriptive text link. Added. Also removed the workflow section of the page.yml, it was left over from the general animation-data page.